spec/administrate/views/fields/has_one/_show_spec.rb

```diff
@@ -23,6 +23,14 @@
     it "renders a link to the record" do
       product = create(:product)
       product_path = polymorphic_path([:admin, product])
+      nested_show = instance_double(
+        "Administrate::Page::Show",
+        resource: double(
+          class: ProductMetaTag,
+        ),
```

Comment:
I think we can provide the real deal here, instead of a stub:

```suggestion
        resource: create(:product_meta_tag),
```